[2023-04-27T07:08:10.532+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag_api.python_task1 manual__2023-04-27T07:08:06.429483+00:00 [queued]>
[2023-04-27T07:08:10.561+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag_api.python_task1 manual__2023-04-27T07:08:06.429483+00:00 [queued]>
[2023-04-27T07:08:10.563+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-27T07:08:10.565+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-27T07:08:10.569+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-27T07:08:10.622+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): python_task1> on 2023-04-27 07:08:06.429483+00:00
[2023-04-27T07:08:10.641+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag_api', 'python_task1', 'manual__2023-04-27T07:08:06.429483+00:00', '--job-id', '127', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag_api.py', '--cfg-path', '/tmp/tmpo7q6h5ds']
[2023-04-27T07:08:10.644+0000] {standard_task_runner.py:83} INFO - Job 127: Subtask python_task1
[2023-04-27T07:08:10.636+0000] {standard_task_runner.py:55} INFO - Started process 772 to run task
[2023-04-27T07:08:11.076+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag_api.python_task1 manual__2023-04-27T07:08:06.429483+00:00 [running]> on host 80d943d8f5b4
[2023-04-27T07:08:11.306+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag_api
AIRFLOW_CTX_TASK_ID=python_task1
AIRFLOW_CTX_EXECUTION_DATE=2023-04-27T07:08:06.429483+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-27T07:08:06.429483+00:00
[2023-04-27T07:08:11.308+0000] {weather_dag_api.py:24} INFO - Starting db creation
[2023-04-27T07:08:11.309+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-27T07:08:11.337+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag_api, task_id=python_task1, execution_date=20230427T070806, start_date=20230427T070810, end_date=20230427T070811
[2023-04-27T07:08:11.416+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-27T07:08:11.492+0000] {dagrun.py:697} ERROR - Failed to get task for ti <TaskInstance: weather_dag_api.cities.view_data_Lviv manual__2023-04-27T07:08:06.429483+00:00 [None]>. Marking it as removed.
[2023-04-27T07:08:11.501+0000] {dagrun.py:697} ERROR - Failed to get task for ti <TaskInstance: weather_dag_api.cities.view_data_Kyiv manual__2023-04-27T07:08:06.429483+00:00 [None]>. Marking it as removed.
[2023-04-27T07:08:11.507+0000] {dagrun.py:697} ERROR - Failed to get task for ti <TaskInstance: weather_dag_api.cities.view_data_Kharkiv manual__2023-04-27T07:08:06.429483+00:00 [None]>. Marking it as removed.
[2023-04-27T07:08:11.512+0000] {dagrun.py:697} ERROR - Failed to get task for ti <TaskInstance: weather_dag_api.cities.view_data_Odesa manual__2023-04-27T07:08:06.429483+00:00 [None]>. Marking it as removed.
[2023-04-27T07:08:11.516+0000] {dagrun.py:697} ERROR - Failed to get task for ti <TaskInstance: weather_dag_api.cities.view_data_Zhmerynka manual__2023-04-27T07:08:06.429483+00:00 [None]>. Marking it as removed.
[2023-04-27T07:08:11.557+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
